litmus-portal/frontend/src/pages/ApplicationDashboard/index.tsx

```diff
@@ -156,219 +280,601 @@ const DashboardPage: React.FC = () => {
-                        value={data.db_id}
-                        onClick={() => {
-                          setSelectedDashboardInformation({
-                            ...selectedDashboardInformation,
-                            id: data.db_id,
-                            name: data.db_name,
-                            typeName: data.db_type_name,
-                            typeID: data.db_type_id,
-                            urlToIcon: `./icons/${data.db_type_id}_dashboard.svg`,
-                            information: data.db_information,
-                            chaosEventQueryTemplate:
-                              data.chaos_event_query_template,
-                            chaosVerdictQueryTemplate:
-                              data.chaos_verdict_query_template,
-                            applicationMetadataMap:
-                              data.application_metadata_map,
-                          });
-                          dataSource.selectDataSource({
-                            selectedDataSourceURL: '',
-                            selectedDataSourceID: '',
-                            selectedDataSourceName: '',
-                          });
-                          setAnchorEl(null);
-                        }}
-                        className={
-                          data.db_id === selectedDashboardInformation.id
-                            ? classes.menuItemSelected
-                            : classes.menuItem
-                        }
-                      >
-                        <div className={classes.expDiv}>
-                          <Typography
-                            data-cy="switchDashboard"
-                            className={`${classes.btnText} ${classes.italic}`}
-                            variant="h5"
-                          >
-                            {data.db_name}
-                          </Typography>
-                        </div>
-                      </MenuItem>
-                    );
-                  }
-                )}
-              </Menu>
-            </Typography>
+                <Typography>
+                  {t('analyticsDashboard.monitoringDashboardPage.goBack')}
+                </Typography>
+              </ButtonFilled>
+            </div>
+          </div>
+        ) : (
+          <div className={classes.root}>
+            <div className={classes.button}>
+              <BackButton />
+            </div>
 
-            <TopNavButtons
-              isInfoToggledState={isInfoOpen}
-              switchIsInfoToggled={(toggleState: Boolean) => {
-                setIsInfoOpen(toggleState);
+            <div className={classes.controlsDiv}>
+              <Typography variant="h4" style={{ fontWeight: 500 }}>
+                {`${selectedDashboardInformation.agentName} / `}
+                <Typography
+                  variant="h4"
+                  display="inline"
+                  style={{ fontStyle: 'italic' }}
```

Comment:
Avoid inline style

Reply:
@whitetiger1399  here we needed to override a single property so it has been done refer to review comments of @Saranya-jena that i addressed.